src/main/java/duke/common/Messages.java

```diff
@@ -0,0 +1,27 @@
+package duke.common;
+
+import static duke.storage.StorageHandler.listInputs;
+
+public class Messages {
+    public static final String BYE_MESSAGE = " " + "Bye. Hope to see you again soon!";
+    public static final String SAVED_MESSAGE = " " + "Nice! I have saved your list.";
+    public static final String LIST_HEADER_MESSAGE = " "  + "Here are the tasks in your list:";
+    public static final String DONE_HEADER_MESSAGE = " " + "Nice! I've marked this task as done: ";
+    public static final String DEADLINE_FORMAT_MESSAGE = " " + "Invalid deadline declaration\n" +
+            " " + "deadline {info} /by {date}";
+    public static final String EVENT_FORMAT_MESSAGE = " " + "Invalid event declaration\n" +
+            " " + "event {info} /by {date} {time}";
+    public static final String INVALID_INPUT_MAIN_MESSAGE =
+            " " + "☹ OOPS!!! I'm sorry, but I don't know what that means :-(";
+    public static final String EMPTY_DONE_INPUT_MESSAGE = " " + "☹ OOPS!!! The description of a done cannot be empty.";
+    public static final String EMPTY_TODO_INPUT_MESSAGE = " " + "☹ OOPS!!! The description of a todo cannot be empty.";
+    public static final String EMPTY_DEADLINE_INPUT_MESSAGE =
+            " " + "☹ OOPS!!! The description of a deadline cannot be empty.";
+    public static final String EMPTY_EVENT_INPUT_MESSAGE =
+            " " + "☹ OOPS!!! The description of a event cannot be empty.";
+    public static final String EMPTY_DELETE_INPUT_MESSAGE =
+            " " + "☹ OOPS!!! The description of a delete cannot be empty.";
```

Comment:
Since each of the constants carry some message text, it is better to name them as `MESSAGE_<something>`
Alternatively, since this class only contains messages, you can eliminate the `MESSAGE` word from the variable name, so that it reads `Messages.BYE`, `Messages.INVALID_INPUT`